code/modules/mob/living/living.dm

```diff
@@ -471,6 +471,10 @@
 
 ///Proc to hook behavior to the change of value in the resting variable.
 /mob/living/proc/set_resting(new_resting, silent = TRUE, instant = FALSE)
+	if(!(mobility_flags & MOBILITY_REST))
+		get_up(instant)
+		update_resting()
+		return
```

Comment:
This should be handled in `/mob/living/proc/toggle_resting()` instead, early returning there without getting up or updating resting.

Reply:
There are direct usages of set_resting in the code, including for placing mobs on tables, which is the bug this is supposed to fix. If I move that to toggle_resting, this PR won't fix the bug. Are you absolutely sure you want me to do this?

Reply:
I dislike it being handled at such low level, there should be checks before, but oh well. It will have to do.